index.html

```diff
@@ -90,7 +90,7 @@ <h2 class="dark-blue">Hey there.I’m Farid <br> <span class="green">I’m a sof
                     </li>
```

Comment:
>  this comment actually refers to line #50: 

- [ ] It would be nice to have one `h1` tag in your file semantically indicating the title of the page. Currently, you only have `h2` tags.